orga/app/components/auth/login-form.js

```diff
@@ -47,12 +47,20 @@ export default class LoginForm extends Component {
           this.args.organizationInvitationCode,
           email
         );
-      } catch (responseError) {
-        responseError.errors.forEach((error) => {
-          if (error.status === '412') {
-            return this._authenticate(password, email);
-          }
-        });
+      } catch (err) {
+        const error = Array.isArray(err.errors) && err.errors.length > 0 && err.errors[0];
+        const isInvitationAlreadyAcceptedByAnotherUser = error && error.status === '409';
+        if (isInvitationAlreadyAcceptedByAnotherUser) {
+          this.errorMessage = this.intl.t('pages.login-form.errors.status.409');
+          this.isLoading = false;
+          return;
+        }
+        const isUserAlreadyOrganizationMember = error && error.status === '412';
+        if (!isUserAlreadyOrganizationMember) {
+          this.errorMessage = this.intl.t(this._getI18nKeyByStatus(error.status));
```

Comment:
Modifier : ajouter la clé pour la 412 dans _getI18nKeyByStatus

Reply:
J'y pense après coup, la 412 n'est bloquante, il devrait pouvoir se connecter vu qu'il fait déjà partie de l'organisation. Côté certif j'avais implémenté ça comme ça https://github.com/1024pix/pix/blob/c7d4ded506e9170e4b759d62272ce98ecaf99757/certif/app/components/auth/toggable-login-form.js#L56

Reply:
Tout à fait, la `412` n'est pas bloquante.

Je n'ai pas voulu mettre la `412` dans `_getI18nKeyByStatus` car cette `412` est spécifique à `_acceptOrganizationInvitation` et n'apparait pas dans le workflow de login. Je ne voudrais pas qu'on puisse penser qu'en faisant un `_authenticate` on peut avoir une `412`.

Reply:
L'authentification n'est fait qu'après cet appel via le ` return this._authenticate(password, email);`
On reçoit bien une 412 avant